hermes-console-vue/src/composables/use-stats/useStats.ts

```diff
@@ -0,0 +1,184 @@
+import { querySubscriptions, queryTopics } from '@/api/hermes-client';
+import { ref } from 'vue';
+import type { Ref } from 'vue';
+
+export interface UseStats {
+  topicCount: Ref<number | undefined>;
+  ackAllTopicCount: Ref<number | undefined>;
+  ackAllTopicShare: Ref<number | undefined>;
+  trackingEnabledTopicCount: Ref<number | undefined>;
+  trackingEnabledTopicShare: Ref<number | undefined>;
+  avroTopicCount: Ref<number | undefined>;
+  avroTopicShare: Ref<number | undefined>;
+  subscriptionCount: Ref<number | undefined>;
+  trackingEnabledSubscriptionCount: Ref<number | undefined>;
+  trackingEnabledSubscriptionShare: Ref<number | undefined>;
+  avroSubscriptionCount: Ref<number | undefined>;
+  avroSubscriptionShare: Ref<number | undefined>;
+  loading: Ref<boolean>;
+  error: Ref<UseStatsErrors>;
+}
+
+export interface UseStatsErrors {
+  fetchTopics: Error | null;
+  fetchAckAllTopics: Error | null;
+  fetchTrackingEnabledTopics: Error | null;
+  fetchAvroTopics: Error | null;
+  fetchSubscriptions: Error | null;
+  fetchTrackingEnabledSubscriptions: Error | null;
+  fetchAvroSubscriptions: Error | null;
+}
+
+export function useStats(): UseStats {
```

Comment:
Looks pretty good, probably could be simplified after adding dedicated endpoint for statistics in management api